models/replay.go

```diff
@@ -0,0 +1,39 @@
+package models
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+const (
+	// ReplayStatusAccepted worker picked up the request
+	ReplayStatusAccepted   = "accepted"
+	ReplayStatusInProgress = "inprogress"
+	// ReplayStatusFailed worker fail while processing the replay request
+	ReplayStatusFailed  = "failed"  // end state
+	ReplayStatusSuccess = "success" // end state
+)
+
+type ReplayMessage struct {
+	Type    string
+	Message string
+}
+
+type ReplayWorkerRequest struct {
+	ID         uuid.UUID
+	Job        JobSpec
+	Start      time.Time
+	End        time.Time
+	Project    ProjectSpec
+	DagSpecMap map[string]JobSpec
```

Comment:
Any reason we are calling this `DagSpec`

Reply:
sure, updated to jobSpecMap